src/cljs/rems/application.cljs

```diff
@@ -60,14 +60,11 @@
  ::enter-application-page
  (fn [{:keys [db]} [_ id]]
    (merge {:db (reset-state db)
-           ::fetch-application [id]}
+           ::fetch-application id}
           (when (contains? (get-in db [:identity :roles]) :approver)
             {::fetch-potential-third-party-reviewers [(get-in db [:identity :user])]}))))
 
-(defn- fetch-application [id]
-  (fetch (str "/api/applications/" id) {:handler #(rf/dispatch [::fetch-application-result %])}))
-
-(rf/reg-fx ::fetch-application (fn [[id]] (fetch-application id)))
+(rf/reg-fx ::fetch-application (fn [id] (fetch (str "/api/applications/" id) {:handler #(rf/dispatch [::fetch-application-result %])})))
```

Comment:
I think this is too long for one line

Reply:
It's 136 characters whereas we have other line with 163 already in the same file.

In this case I think it's simple enough code that it may pass.